pros/ga/analytics.py

```diff
@@ -48,17 +51,20 @@ def send(self,action):
                 'el': 'CLI',
                 'ev': '1',
                 'ni': 0
-            }          
+            }
+
+            session = FuturesSession()          
 
             #Send payload to GA servers 
-            response = requests.post(url=url,
+            future = session.post(url=url,
                              data=payload,
                              headers={'User-Agent': agent},
                              timeout=5.0)
-            if not response.status_code==200:
-                print("Something went wrong while sending analytics!")
-                print(response)
-            return response
+            self.pendingRequests.append(future)
```

Comment:
make sure to deleted these commented lines

Reply:
just removed the lines and pushed, should be good to go